loopchain/channel/channel_inner_service.py

```diff
@@ -76,8 +81,11 @@ def cleanup(self):
             ts = TransactionSerializer.new(tx_version, self.__tx_versioner)
             tx = ts.from_(kwargs)
 
+            if self.__nid is None:
```

Comment:
Isn't there any id in kwargs?
I think it's better to check nid first before allocate TransactionSerializer.